app/assets/javascripts/registration/grid.js.erb

```diff
@@ -470,6 +470,17 @@ var gridContext = function() {
           }
         })
 
+        // Show/hide external workflow drop-down when workflow selector changes
```

Comment:
I think the test could go here: https://github.com/sul-dlss/argo/blob/ac6483915e12ca0cfe4735ca8bab907d962e97c7/spec/features/item_registration_spec.rb

As far as examples for testing that form specifically.. no. In the Argo work last year we didn't touch this stuff, thus it didn't get tests added.

An example of testing javascript forms with Capybara/Poltergeist: https://github.com/sul-cidr/histonets/blob/master/spec/features/image_clean_spec.rb#L24-L40

not the cleanest example there, but the most recent.